src/store/cesium/useCesium.ts

```diff
@@ -0,0 +1,333 @@
+
+import * as Cesium from 'cesium';
+import { GeoJsonDataSource } from 'cesium';
+import { ref, watch, Ref } from 'vue';
+import { GeoJSON } from 'geojson';
+import { rgdFootprint, rgdRegionSites } from '@/api/rest';
+
+
+type Degrees = {
+  longitude: number,
+  latitude: number,
+  height: number,
+};
+
+export default function useCesium(element?: Ref<HTMLElement | null>) {
```

Comment:
```suggestion
export default function useCesium() {
```
This is reported as a warning by `yarn storybook`

Reply:
resolved with 38eb4c6fb645448f9e4199772d46bfbca8cac6fa